### DISCUSSION TITLE
Lesson 11 - encountered error while deploying

### DISCUSSION THREAD

**Discussion by lpetroulakis** [Cyfrin/foundry-full-course-cu#1100] [Q&A] [answered]
Greetings fellow developers!
while doing lesson 11 and after copying and pasting the repo's makefile, i have the following deployable script
```solidity
pragma solidity ^0.8.18;

import {Script} from "forge-std/Script.sol";
import {BasicNft} from "../src/BasicNft.sol";

contract DeployBasicNft is Script {
    function run() external returns (BasicNft) {
        vm.startBroadcast();
        BasicNft basicNft = new BasicNft();
        vm.stopBroadcast();
        return basicNft;
    }
}
```
however, when i execute the following in the terminal
```
make deploy ARGS="--network sepolia" 
```
i get the following error
```
ONCHAIN EXECUTION COMPLETE & SUCCESSFUL.
Total Paid: 0.0029817600099392 ETH (993920 gas * avg 3.00000001 gwei)
##
Start verification for (1) contracts
Start verifying contract `0x02581Cf12e5FE2596e1E8a26F61136951c7D2b08` deployed on sepolia

Transactions saved to: /home/lpetroulakis/nft-23/broadcast/DeployBasicNft.s.sol/11155111/run-latest.json

Sensitive values saved to: /home/lpetroulakis/nft-23/cache/DeployBasicNft.s.sol/11155111/run-latest.json

Error: 
expected value at line 1 column 1
make: *** [Makefile:43: deploy] Error 1
```
everything seems to be going fine until the verification step when this particular error pops up, no matter what i've tried with google, AI, and similar discussions here. Any help would be much appreciated!

Cheers!

btw, line 43 in my makefile is the following
```solidity
	@forge script script/DeployBasicNft.s.sol:DeployBasicNft $(NETWORK_ARGS)
```

**Comment by 0xusmanf** [comment]
Have you updated `ETHERSCAN_API_KEY` value in your `.env` file?

**Reply by lpetroulakis** [reply]
Yeah, first thing i checked! Thank you for the reply!

**Reply by 0xusmanf** [reply, marked as answer]
Check [here](https://github.com/Cyfrin/foundry-full-course-f23/discussions/276), your error indicates there is something wrong with your API KEY.

**Reply by lpetroulakis** [reply]
There seemed to be a problem with my API key, i was pasting it correctly but it did no good. I generated a new one and it worked fine! Thanks a lot mate!

**Reply by 0xusmanf** [reply]
You are welcome, happy to help.
